### ISSUE TITLE
User rank is not point-in-time in User Matches end-point

### ISSUE THREAD

**Issue by souzatharsis** [ForFunLabs/webapi#4]
The user rank in the 'user matches' end-point is not point-in-time, i.e., the rank value in the user attribute of a match shows the most recent rank of the user instead of the user rank at the time of the match.

This issue is possibly also present in the matches API. TBD.

**Comment by oscarvazquez** [comment]
This is correct unfortunately we weren't recording the user rank before the match. We do plan on adding it so I'll mark this as an improvement idea and leave it open
